### DISCUSSION TITLE
Cache(Static) and Cache(Dynamic)...which is best to use ?

### DISCUSSION THREAD

**Discussion by Arun10752** [Xanashi/Icaros#92] [General]
which setting will save cache more permanent and loads thumbnails superfast ? currently i use cache(Dynamic)

**Comment by Xanashi** [comment]
Both cache settings have the same benefit of speeding up the thumbnailing in Explorer, the only difference is that the
dynamic mode will continuously add any new thumbnail to the cache, while the static mode will keep the cache at it's current size, but still
allow Icaros to use it to speed up thumbnailing of any file that has already been cached. 

Static should only be used if you have limited space or don't want certain files to be cached in Icaros. (Could be the case in corporate settings). 

It is still possible to add and remove files from the cache using the Indexer in IcarosConfig, while the static mode is selected,
allowing users to have better control over what files are being cached. 

For most users the dynamic mode is the way to go though.

There is more info here:
https://github.com/Xanashi/Icaros?tab=readme-ov-file#icaros-cache
I plan to update it to be more understandable as soon as I have the time.

**Reply by Arun10752** [reply]
Thanks for the Guide.
